CODING_STANDARDS.md

```diff
@@ -203,6 +203,28 @@ use these rules.
     RADAR // Radio Detecting and Ranging
     ```
 
+## User function/method parameter naming conventions
+
+1. Function parameter names for user-level functions should be in lowercase,
+    with words underscore delimited, with care taken to minimize the letter
+    count. Abbreviations should not be used when they greatly decrease the
+    readability of the parameter.
+
+2. Method parameter names follow the *studlyCaps* (also referred to as
+    *bumpy case* or *camel caps*) naming convention, with care taken to minimize
```

Comment:
```suggestion
    *bumpy case* or *camel case*) naming convention, with care taken to minimize
```

Reply:
Thanks for reviewing. The term _camel caps_ is already used elsewhere in `CODING_STANDARDS.md`, so I kept things consistent. While both _camel caps_ and _camel case_ are correct you are right that _camel case_ is more commonly used.